### PR TITLE
docs: creates README.md

### PR DESCRIPTION
**Pull request by kabir**
Suggested basic readme as part of a WildFly community building exercise

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+= Shrinkwrap
```
Comment (bartoszmajsak) [inline README.md:1]:
```suggestion
# Shrinkwrap
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+ShrinkWrap is a library to create archives from Java, which powers the [Arquillian]([url](https://github.com/arquillian))
```
Comment (bartoszmajsak) [inline README.md:3]:
```suggestion
ShrinkWrap is a library to create archives from Java, which powers the [Arquillian](https://github.com/arquillian)
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+= Shrinkwrap
+
+ShrinkWrap is a library to create archives from Java, which powers the [Arquillian]([url](https://github.com/arquillian)) 
+deployment mechanism. Arquillian in turn is a library.
+
+Some information on how to use Shrinkwrap to create archives with Arquillian can be found in this [guide[(https://arquillian.org/guides/shrinkwrap_introduction/)
```
Comment (bartoszmajsak) [inline README.md:6]:
```suggestion
Some information on how to use Shrinkwrap to create archives with Arquillian can be found in this [guide](https://arquillian.org/guides/shrinkwrap_introduction/)
```

**Review by bartoszmajsak — CHANGES_REQUESTED** [review]
Thanks for putting this together. There are a few mistakes in markdown syntax. I added suggestions for ease of correcting it all at once.

**Comment by kabir** [comment]
@bartoszmajsak Thanks for the suggestions, I've incorporated them. This was all done in a bit of a rush as you can probably tell :-)

**Review by bartoszmajsak — APPROVED** [review]

**Pull request merged**